src/ConvertMD.ts

```diff
@@ -0,0 +1,206 @@
+import * as fs from 'fs';
+import * as yaml from 'js-yaml';
+
+interface AsyncAPIDocument {
```

Comment:
do not create/maintain this interface, use @asyncapi/js-parser for this, of just use <any>